README.md

```diff
@@ -45,14 +45,24 @@
   - [Input Output Model](#input-output-model)
   - [Circular Economy and Waste](#circular-economy-and-waste)
 - [Biosphere](#biosphere)
-  - [Biodiversity and Species Distribution](#biodiversity-and-species-distribution)
+  - [Biodiversity Data Access and Management](#biodiversity-data-access-and-management)
+  - [Biodiversity Data Cleaning and Standardization](#biodiversity-data-cleaning-and-standardization)
+  - [Species Distribution Modeling](#species-distribution-modeling)
+  - [Biodiversity Analysis and Metrics](#biodiversity-analysis-and-metrics)
+  - [Ecological and Environmental Modeling](#ecological-and-environmental-modeling)
+  - [Biodiversity Citizen Science](#biodiversity-citizen-science)
   - [Conservation and Restoration](#conservation-and-restoration)
+  - [Deforestation and Reforestation](#deforestation-and-reforestation)
+  - [Forest Remote Sensing](#forest-remote-sensing)
   - [Forest Observation and Management](#forest-observation-and-management)
+  - [Forest Modeling and Analysis](#forest-modeling-and-analysis)
   - [Plants and Vegetation](#plants-and-vegetation)
   - [Biomass](#biomass)
   - [Wildfire](#wildfire)
+  - [Terrestrial Wildlife](#terrestrial-wildlife)
+  - [Avian Monitoring and Analysis](#avian-monitoring-and-analysis)
+  - [Bioacoustics and Acoustic Data Analysis](#bioacoustics-and-acoustic-data-analysis)
   - [Marine Life and Fishery](#marine-life-and-fishery)
-  - [Terrestrial Animals](#terrestrial-animals)
```

Comment:
I think it would help if the subcategories are listed alphabetically. This would make it way easier to browse in my oppinion.

Reply:
If we use alphabetic we need to do the same for the whole project.  So far I tried to order the categories so that "similar" categories can be found next to the category. 

@RichardLitt Do you think alphabetic ordering of the subcategories makes sense? Your linguistic skills could provide relevant feedback here.  I'm not sure myself what the best approach is here and what is easiest for a user to grasp.

Reply:
I think it could be just on the subcategory level (and maybe category as well) wichout touching the projecst itself.